dev/com.ibm.ws.security.javaeesec.cdi/src/com/ibm/ws/security/javaeesec/cdi/beans/FormAuthenticationMechanism.java

```diff
@@ -162,12 +174,14 @@ private AuthenticationStatus validateWithUserRegistry(Subject clientSubject, @Se
             } catch (Exception e) {
                 throw new AuthenticationException(e.toString());
             }
+        } else {
+            // this condition should not happen.
```

Comment:
If it does not happen, then it could be removed.

Reply:
I'm not 100% whether this condition could happen or not and also it would be normal, so for now, instead of throwing the exception, log the debug message.